cmd/juju/storage/add.go

```diff
@@ -74,13 +74,7 @@ Examples:
     or
       juju add-storage u/0 data 
 `
-	addCommandAgs = `
-<unit name> <storage directive> ...
-    where storage directive is 
-        <charm storage name>=<storage constraints> 
-    or
-        <charm storage name>
-`
+	addCommandAgs = `<unit name> <charm storage name>=[<storage constraints>]`
```

Comment:
does this still put [options] when you run "juju add-storage --help "?

Reply:
Yes, options are still there: http://pastebin.ubuntu.com/25864651/